resources/publications.md

```diff
@@ -95,12 +93,57 @@ td:nth-child(2) {
 
 ## Metrics
 
+Machine translation metrics automatically assess quality of the machine translation output.
+There are two types of metrics: **quality evaluation**  and **quality estimation**.   
+* [Quality evaluation](/quality/quality-evaluation.md) metrics rely on human (reference) translation.
+* [Quality estimation](/quality/quality-estimation.md) metrics ***do not*** rely on human (reference) translation. 
+
+### Quality evaluation
+
+#### Similarity-based metrics
+
+These metrics evaluate similarity between machine translation and reference translation. This similarity can be based on:
```

Comment:
We've realized that incomplete introductory sentences for lists are not machine translation-friendly.  We've added it to the [style guide](https://machinetranslate.org/style#lists).

Maybe a good option here could be something like this (please fix if my phrasing is incorrect).

> These similarities can be based on two matching patterns:

Reply:
Dear Cecilia,
Thank you for finding these! Updated.